libs/langchain-google-common/src/utils/common.ts

```diff
@@ -41,6 +41,11 @@ export function copyAIModelParamsInto(
     options?.convertSystemMessageToHumanContent ??
     params?.convertSystemMessageToHumanContent ??
     target?.convertSystemMessageToHumanContent;
+  ret.responseMimeType =
+    options?.responseMimeType ??
+    params?.responseMimeType ??
+    target?.responseMimeType ??
+    "text/plain";
```

Comment:
Rather than put the default here, we tend to have the defaults specified in the class that implements it (ie - `GoogleBaseLLM` and `ChatGoogleBase`.